File: combine_arrays.js
function combineArrays(arr1, arr2) {
    let combinedArray = [];

    let i = j = 0;

    while (i < arr1.length && j < arr2.length) {
        if (arr1[i] < arr2[j]) {
            combinedArray.push(arr1[i]);
            i++;
        } else {
            combinedArray.push(arr2[j]);
            j++
        }
    }
    while (i < arr1.length) {
        combinedArray.push(arr1[i]);
        i++;
    }
    while (j < arr2.length) {
        combinedArray.push(arr2[j])
        j++;
    }
    return combinedArray
}

console.log(combineArrays([1, 2, 5, 8], [8, 12, 14, 67]));

File: Arrays_n_Strings/combine_arrays.py
# This functionn takes in two sorted arrays

def combineTwoArrays(arr1, arr2):
    combinedArray = []

    i = j = 0

    while i < len(arr1) and j < len(arr2):
        if arr1[i] < arr2[j]:
            combinedArray.append(arr1[i])
            i += 1
        else:
            combinedArray.append(arr2[j])
            j += 1

    while i < len(arr1):
        combinedArray.append(arr1[i])
        i += 1

    while j < len(arr2):
        combinedArray.append(arr2[j])
        j += 1

    return combinedArray


print(combineTwoArrays([6, 7, 23, 67], [3, 77, 103, 549]))